### PR TITLE
Build using msbuild, update submodules

### PR DESCRIPTION
**Pull request by dundee**

### CODE REVIEW

File: ironpython-git/PKGBUILD [hunk 1]
```diff
@@ -1,19 +1,17 @@
 # Maintainer: Michael Schubert <mschu.dev at gmail>
 
 pkgname=ironpython-git
-pkgver=2.7.2.a2.471.g7be8b73
+pkgver=ac72964d7
```
Comment (mschubert) [inline ironpython-git/PKGBUILD]: `pkgver` is not valid here. It should be `2.7.7.ac72964d7` (details [here](https://wiki.archlinux.org/index.php/VCS_package_guidelines))

File: ironpython-git/PKGBUILD [hunk 2]
```diff
@@ -1,19 +1,17 @@
...
 pkgrel=1
 pkgdesc="Python implementation for the .NET framework"
 arch=("any")
 url="http://ironpython.net"
 license=("Apache")
-depends=('mono')
+depends=('mono' 'msbuild-bin')
```
Comment (mschubert) [inline ironpython-git/PKGBUILD]: This should be in `makedepends`, not `depends`, if it's just used to build the software but not required to run it.

File: ironpython-git/PKGBUILD [hunk 3]
```diff
@@ -22,8 +20,10 @@ pkgver() {
 
 build() {
   cd "$srcdir/$pkgname"
-  sed -i "/NoWarn/s|444;||" Solutions/Common.proj
-  xbuild Solutions/IronPython.sln /p:Mono=true /p:Configuration="Release"
+  export LC_ALL="en_US.UTF-8"
```
Comment (mschubert) [inline ironpython-git/PKGBUILD]: Is this (and `LANG=`) required? User environments should be respected where possible, so if we can do it without forcing a US locale that would be great

**Review by mschubert — COMMENTED** [review]
Hi,
Thank you very much for the patch.
I made a couple of comments, if you have time please have a look (otherwise I'll edit them myself tomorrow or on Monday)

File: ironpython-git/PKGBUILD (hunk 1 above)
Comment (dundee) [reply ironpython-git/PKGBUILD]: Then we should probably change pkgver function so it updates this variable correctly.

File: ironpython-git/PKGBUILD (hunk 2 above)
Comment (dundee) [reply ironpython-git/PKGBUILD]: Good point!

File: ironpython-git/PKGBUILD (hunk 3 above)
Comment (dundee) [reply ironpython-git/PKGBUILD]: I had troubles building when using xbuild without this setting. Now it looks ok without it.

**Comment by dundee** [comment]
Hi, is it all right now?

**Pull request merged**